classes/core-image-pelux.bbclass

```diff
@@ -52,6 +52,11 @@ IMAGE_INSTALL_remove_qemux86-64 = "\
     swupdate \
 "
 
+# Connectivity Manager
+IMAGE_INSTALL_append = "\
```

Comment:
Will it be possible to test connectivity-manager on qemu? If not then may be remove from qemu.

Reply:
To test WiFi itself one would have to set that up in QEMU. Do not know how to do that. But should still be possible to run it. And if we add e.g. ethernet that should be possible to test.